clmm/utils.py

```diff
@@ -22,11 +22,11 @@ def compute_nfw_boost(rvals, rs=1000, b0=0.1) :
 
     Returns
     -------
-    boost_factors : array_like
+    boost_factors : numpy.ndarray
```

Comment:
now these arguments are always being converted to np.arrays, so I guess the old description was better, no?

Reply:
I thought here was for the output of the function. Output of the function is always np array regardless of the input type.

Reply:
you are right